package.json

```diff
@@ -24,7 +24,7 @@
     "lint": "npm --prefix src/brave run lint --",
     "test": "npm --prefix src/brave run test --",
     "test:scripts": "jest lib scripts --passWithNoTests",
-    "test-security": "npm run audit_deps && node ./scripts/commands.js start --enable_brave_update --network_log --user_data_dir_name=brave-network-test"
+    "test-security": "npm run audit_deps -- --audit_dev_deps && node ./scripts/commands.js start --enable_brave_update --network_log --user_data_dir_name=brave-network-test"
```

Comment:
it could be `npm --prefix src/brave run test-security --` but not sure that would audit `package.json` here in b-b

Reply:
audit_deps in brave-core audits the brave-browser directory by default so that would work

Reply:
@mihaiplesa done!

Reply:
let's see what CI says